code/WorkInProgress/Cael_Aislinn/Rust/core_gen.dm

```diff
@@ -112,6 +129,7 @@ max volume of plasma storeable by the field = the total volume of a number of ti
 			owned_field.ChangeFieldFrequency(field_frequency)
 
 /obj/machinery/power/rust_core/proc/Startup()
+	light_color = LIGHT_COLOR_BLUE
```

Comment:
You don't need to ever turn light color off, if either light range or light power is 0 no light is made anyways.